.github/workflows/run-integration-tests.yml

```diff
@@ -11,8 +11,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  all-integration-tests:
-    name: 'Run Aurora integration tests'
+  all-integration-tests-latest:
```

Comment:
Integration may run for hours and sometimes it reach the max allowed run time of 6h. Increasing  (doubling) amount of work for this github action will definitely increase the chance of such timeout out event. I'd suggest to split this script into two separate github actions.

Reply:
Adding to what @sergiyvamz said, another way to run multiple jobs in parallel without duplicating code is to use matrix https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs